Makefile

```diff
@@ -1,6 +1,6 @@
 CC := $(CC)
 CXX := $(CXX)
-CXXFLAGS := $(CXXFLAGS) -Iinclude -Imason_packages/.link/include -std=c++11
+CXXFLAGS := $(CXXFLAGS) -Iinclude -Imason_packages/.link/include -I/usr/local/include -std=c++11
```

Comment:
@ericfischer Can you say why you added`-I/usr/local/include`? Ideally this should not be necessary?